.github/workflows/ci.yml

```
@@ -193,6 +237,40 @@ jobs:
+  # slips into the main path and breaks Windows/macOS without anyone noticing
+  # until release time. `release-build` exists but only verifies what
+  # goreleaser's `goos:` happens to enumerate; if a future PR drops `windows`
+  # from .goreleaser.yml, this is the gate that catches it independently.
+  go-cross-build:
+    needs: web
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { goos: windows, goarch: amd64 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)